internal/cmd/controller/reconciler/bundle_controller.go

```diff
@@ -216,38 +232,25 @@ func (r *BundleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	summary.SetReadyConditions(&bundle.Status, "Cluster", bundle.Status.Summary)
 	bundle.Status.ObservedGeneration = bundle.Generation
 
-	// build BundleDeployments out of targets discarding Status, replacing
-	// DependsOn with the bundle's DependsOn (pure function) and replacing
-	// the labels with the bundle's labels
+	// build BundleDeployments out of targets discarding Status, replacing DependsOn with the
+	// bundle's DependsOn (pure function) and replacing the labels with the bundle's labels
 	for _, target := range matchedTargets {
 		bd, err := r.createBundleDeployment(ctx, logger, target, contentsInOCI, manifestID)
 		if err != nil {
 			return ctrl.Result{}, err
 		}
 
 		if bd != nil && contentsInOCI {
-			// we need to create the OCI registry credentials secret in the BundleDeployment's namespace
+			// we need to create the OCI registry credentials secret in the BundleDeployment's
+			// namespace
 			if err := r.createDeploymentOCISecret(ctx, bundle, bd); err != nil {
 				return ctrl.Result{}, err
 			}
 		}
 	}
 
 	updateDisplay(&bundle.Status)
-
-	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
No more retries on conflicts? Is that deliberate?
Same question about other instances below.

Reply:
Yes, it is deliberate! We've started migrating away from RetryOnConflict and let controller-runtime's back-off do more of the retries, just a bit further apart. Those were some of the few leftovers of RetryOnConflict and I was asked to take care of it while I'm already working on those controllers.

Reply:
Oh, that does ring a bell. Thanks for clarifying.